builder/variants/defaults-gpl.sh

```diff
@@ -6,3 +6,5 @@ GIT_BRANCH="jellyfin"
 LICENSE_FILE="COPYING.GPLv3"
 
 [[ $TARGET == linux* ]] && FF_CONFIGURE+=" --disable-libxcb --disable-xlib" || true
+[[ $TARGET == win* ]] && FF_CFLAGS+=" -Wa,-muse-unaligned-vector-move" || true
```

Comment:
https://github.com/jellyfin/jellyfin-ffmpeg/blob/4edf5f899ead175d22ee9ce8a56604365b940e92/docker-build-win64.sh#L591

The non ct-ng builder also requires Wa.
```
CFLAGS+=" -Wa,-muse-unaligned-vector-move"
CXXFLAGS+=" -Wa,-muse-unaligned-vector-move"

./configure ...
```
It seems that there is no easy way to add the flag only for specific files...

Reply:
Isn't this file also used by ct-ng?

If so we should add this flag exclusively to this file instead of defaults-gpl.sh

Reply:
Nope. ct-ng is scoped to `./builder`.

`docker-build-win64.sh` is a separate builder that uses Ubuntu's mingw instead of building the toolchain from scratch.